src/syncData.ts

```diff
@@ -76,13 +79,20 @@ export const initSync = (
   ) => {
     debug('Change stream result %O', result)
     if ('rowcount' in result) {
-      emit('process', {
+      const event = {
         success: result.rowcount,
+        fail: 0,
```

Comment:
Am I understanding right that this is effectively the only change? Everything else looks like it would only affect TypeScript compilation.